apps/client/src/routes/repositories/+page.ts

```diff
@@ -34,7 +34,10 @@ export const load = async (event) => {
       },
     });
     const repositories = await githubClient.query({
-      query: FETCH_REPOSITORIES_BY_TOPIC,
+      query: FETCH_REPOSITORIES_BY_TOPICS,
+      variables: {
+        topics: "react,flutter,node",
```

Comment:
Don't forget to fetch this from the DB. Also, adding a component to prompt users to subscribe to tags when they aren't subscribed to any would be a great enhancement!

Reply:
Yeah, I plan to fetch them from the userTopics field. This was a test drive